modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding-upgrade-viability-conditiontype_{context}"]
+= Understanding upgrade viability with conditionType
```

Comment:
It's weird to see conditionType not being marked up as monospace (`conditionType`). Maybe work around it with `= Understanding cluster operator condition types` and just leave the mention about impact to the paragraph?